DALI_DEPS_VERSION

```diff
@@ -1 +1 @@
-345c534814ffcfd40c2df3fe3d4a00c0d911b94c
+FIXME
```

Comment:
Revert to what's in main - the relevant DALI_deps PR has already been merged.

Reply:
Done